README.md

```diff
@@ -25,6 +25,10 @@ These code samples are currently based on:
    - If Maven is not downloaded, download it from [here](https://maven.apache.org/download.cgi)
    - For installation, follow the instructions [here](https://maven.apache.org/install.html)
 
+ 3. Gradle
```

Comment:
maven and gradle should be downloaded alternatively right and presence of both is not required. Can we update th readme accordingly.